fhir-smart/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- THIS PRODUCT CONTAINS RESTRICTED MATERIALS OF IBM 5724-H88, 5724-J08,
+    5724-I63, 5655-W65, COPYRIGHT International Business Machines Corp., 2014
+    All Rights Reserved * Licensed Materials - Property of IBM US Government
+    Users Restricted Rights - Use, duplication or disclosure restricted by GSA
+    ADP Schedule Contract with IBM Corp. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.ibm.fhir</groupId>
+        <artifactId>fhir-parent</artifactId>
+        <version>4.3.2-SNAPSHOT</version>
```

Comment:
```suggestion
        <version>4.4.0-SNAPSHOT</version>
```

Reply:
Sure, I can accept your other PR, rebase, and then make this change.